internal/configs/ingress.go

```diff
@@ -313,6 +313,7 @@ func addSSLConfig(server *version1.Server, owner runtime.Object, host string, in
 	}
 
 	var pemFile string
+	var rejectHandshake bool
```

Comment:
Consider assigning `rejectHandshake` as 'true' and handle the single case it's false? If I'm reading it right.

Reply:
it's 2 cases for false vs 3 cases for true